src/utils/AudioManager.ts

```diff
@@ -380,9 +376,9 @@ function settingToGain(value: number) {
     return 0
   const valueRange = 10 // from 0 to 10. 0 => no sound.
   const dbRange = 25 // from -25dB to 0dB. -25 not used (volume=0 => no sound).
-  const normalizedValue = value/valueRange
-  const dB = normalizedValue*dbRange - dbRange
-  return Math.pow(10, dB/20)
+  const normalizedValue = value / valueRange
+  const dB = normalizedValue * dbRange - dbRange
+  return Math.pow(10, dB / 20)
```

Comment:
sorry my auto formatter strike again, do you have a config file for the formatting?

Reply:
We don't. It might be something we should look into